features/routing/ipfailover.feature

```diff
@@ -0,0 +1,29 @@
+Feature: Testing ipfailover scenarios
+
+  # @author hongli@redhat.com
+  # @case_id OCP-9767
+  @admin
+  Scenario: Configure a highly available network service
+    Given the cluster is running on OpenStack
+    And I switch to cluster admin pseudo user
+    And I use the "default" project
+    And default router image is stored into the :router_image clipboard
+    And SCC "privileged" is added to the "ipfailover" service account
```

Comment:
why do we need this? Should users update this service account to use ipfailover? Can't things work OOB somehow?

I'm also concerned about other scenarios relying on this service account having default permissions to collide if executed in parallel.

Reply:
The ”ipfailover" is the default service account when creating IP Failover configuration, and I'm sure it is used by IP Failover feature only.
The SA need sufficient privileges to create ipfailover, or it will prompt user to grant. See below:
```
# oc adm ipfailover -h
<---snip--->
Options:
      --service-account='ipfailover': Name of the service account to use to run the ipfailover pod.
<---snip--->
# oc adm ipfailover ipfailover --virtual-ips="1.2.3.4,5.6.7.8"
error: ipfailover could not be created; service account "ipfailover" does not have sufficient privileges, grant access with: oc adm policy add-scc-to-user privileged -z ipfailover
```

Reply:
It is a global service account. If you change it, then another test case may want to use it at the same time and we need to make test case destructive.

As a user I'd expect default SA to have enough permissions. If dev team insists it should NOT have such by default, then perhaps we can create a special temporary service account and use it instead of the default one, to avoid collision between test scenarios.

WDYT?

Reply:
That make sense. I marked it as destructive. Thanks.